.travis/custom_release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+set -x
+
+#!/bin/bash
+set -e
+set -x
```

Comment:
do we need this twice or just a typo?

Reply:
Typo! we don't need it twice. removing it